File: languages/python/bloock/entity/encryption/decrypter_args.py
import bloock._bridge.proto.encryption_entities_pb2 as proto


class DecrypterArgs:
    def __init__(self, key: str) -> None:
        self.key = key

    def to_proto(self) -> proto.DecrypterArgs:
        return proto.DecrypterArgs(key=self.key)

File: languages/php/Bloock/Entity/Encryption/DecrypterArgs.php
<?php

namespace Bloock\Entity\Encryption;

class DecrypterArgs
{
    public $key;

    public function __construct(string $key)
    {
        $this->key = $key;
    }

    public function toProto(): \Bloock\DecrypterArgs
    {
        $p = new \Bloock\DecrypterArgs();
        $p->setKey($this->key);
        return $p;
    }
}